### ISSUE TITLE
Specific failed text-case reasons are missing from output xml

### ISSUE THREAD

**Issue by Macadoshis** [martinmcwhorter/karma-nunit2-reporter#3]
As for issue https://github.com/jlogar/karma-nunit-reporter/issues/2 :

The xml is valid, but it is missing the specific failed text-case reasons.

As a result Jenkins doesn't recognize a failed test as a fail but as a success...

**Comment by martinmcwhorter** [comment]
Yes. This is a long standing issue I have just lived with. This needs to be fixed. I'll try to get to this this week.

**Comment by Kurtz1993** [comment]
Any updates on this issue?

**Comment by martinmcwhorter** [comment]
No. To be honest I have really dropped the ball here. My personal need for the karma-nunit-reporter has evaporated. I don't use it anymore. 

Mine was very much just a fork of the JUnit reporterter. I just hacked the code until it spit out NUnit like XML. Unfortunately things like the specific test cases are broken. 

If someone would like to take over this repo and the NPM I would be happy to transfer it.

**Comment by Kurtz1993** [comment]
Thank you for your response. Altough I'm not really familiar with NUnit format, I'll try to take a look to this code and check if I can fix it.
